.github/workflows/pull_requests.yaml

```diff
@@ -120,6 +120,17 @@ jobs:
         run: .github/annotate_kind_nodes.sh chart-testing
         if: steps.list-changed.outputs.changed == 'true'
 
+      - name: Create tls helm templates
+        run: |
+          .github/create_tls.sh
+          .github/create_tls.sh "random-domain"
+
+      - name: Move tls files to template
+        run: |
+          mv external-tls-secret.yaml ./charts/redpanda/templates/
+          mv internal-tls-secret.yaml ./charts/redpanda/templates/
+          cp .github/external-service.yaml ./charts/redpanda/templates/
```

Comment:
```suggestion
      - name: Create tls helm templates
        run: |
          .github/create_tls.sh
          .github/create_tls.sh "random-domain"
        if: steps.list-changed.outputs.changed == 'true'

      - name: Move tls files to template
        run: |
          mv external-tls-secret.yaml ./charts/redpanda/templates/
          mv internal-tls-secret.yaml ./charts/redpanda/templates/
          cp .github/external-service.yaml ./charts/redpanda/templates/
        if: steps.list-changed.outputs.changed == 'true'
```

Reply:
Sounds good, out of curios what does this guarantee? I have tried searching but found little.